sormas-ui/src/main/java/de/symeda/sormas/ui/utils/ExternalJournalUtil.java

```diff
@@ -47,11 +48,27 @@
 	 * If the person is not registered in the external journal, a create account/register button is returned
 	 * If the person is registered, a button is returned which opens a popup with further options.
 	 *
+	 * The button is:
+	 * 	- enabled if the associated sormasToSormasEntity(case or contact) is not shared or it is shared and the current SORMAS instance has ownership
```

Comment:
What happens when there are multiple contacts/cases for a person, and one is shared with ownership handed over while the others are not? Is that currently possible via S2S?

Reply:
Indeed, there are multiple scenarios in which there can be one case/contact without ownership and another with ownership for the same person. This indeed creates an inconsistent state for the person as both systems can alter the person data, including the external journal status.

I can think of two ways to reach that state:
* have two cases/contacts for the same person and share one with ownership and the other without
* share a case/contact without ownership and create another case/contact for the same person on the receiving instance

Those situations seem unlikely to happen though. Is this something we should treat? If so, maybe we should take care of the handling for the whole person data form.

Reply:
What about having two contacts for one person, and only sharing one?

Reply:
Indeed, that is also possible and will lead to the same situation. However, in case the person is already registered in the external journal, sharing any associated contact will not be possible.

Reply:
Good point. So this problem is out of scope for this PR.

Reply:
Related ticket created: #5650